telegram/src/main/kotlin/com/github/kotlintelegrambot/network/ApiClient.kt

```diff
@@ -179,6 +182,20 @@ internal class ApiClient(
             secretToken = secretToken?.toMultipartBodyPart(ApiConstants.SetWebhook.SECRET_TOKEN),
         )
 
+        is ByInputStream -> service.setWebhookWithCertificateAsFile(
+            url = url.toMultipartBodyPart(ApiConstants.SetWebhook.URL),
+            certificate = certificate.let {
+                // set media type if not set (one would probably want assert or override here idk)
+                if(it.contentType == null) it.copy(contentType = MediaTypeConstants.UTF_8_TEXT.toMediaType())
+                else it
```

Comment:
I've done this in an attempt to reduce code copying. See the comment for actual concern